hieradata/common.yaml

```diff
@@ -1122,8 +1115,6 @@ monitoring::checks::smokey::features:
     feature: smartanswers
   check_static_mirrors:
     feature: mirror
-  check_tariff:
-    feature: tariff
```

Comment:
This is the subset of Smokey tasks that we create Icinga checks for - the less important Smokey tasks only run in Jenkins.